decidim-participatory_processes/app/queries/decidim/participatory_processes/visible_participatory_processes.rb

```diff
@@ -12,6 +12,7 @@ def query
         processes = Decidim::ParticipatoryProcess.all
```

Comment:
I know this isn't related to your PR, but maybe we should also scope this to the user's organization.

Reply:
@isaacmg410 told me that the scoping is done in another place as a first step for all the queries:
https://github.com/decidim/decidim/blob/f35cea6a2a470cdfc8ac52bace85d654e4790a61/decidim-assemblies/app/queries/decidim/assemblies/organization_published_assemblies.rb#L14
https://github.com/decidim/decidim/blob/f35cea6a2a470cdfc8ac52bace85d654e4790a61/decidim-assemblies/app/queries/decidim/assemblies/organization_assemblies.rb#L12

The same system is being used with participatory processes and conferences.